egs/wsj/s5/utils/reverse_lm.sh

```diff
@@ -38,25 +38,13 @@ mkdir -p $outdir
 for f in phones.txt words.txt L.fst L_disambig.fst phones/; do
   cp -r $langdir/$f $outdir
```

Comment:
I think I should leave the `<s>` and `</s>` filters. I am less keen on leaving the whole script alone, as that would hold the last references to `utils/eps2disambig.pl`, `utils/s2eps.pl`, and `utils/remove_oovs.pl`, all of which are not really useful. Same thing--no WSJ data here though. :(